common/src/main/java/com/microsoft/identity/common/internal/util/BrokerProtocolVersionUtil.java

```diff
@@ -42,4 +43,15 @@ public static boolean canCompressBrokerPayloads(@Nullable String negotiatedBroke
                 MSAL_TO_BROKER_PROTOCOL_COMPRESSION_CHANGES_MINIMUM_VERSION);
 
     }
+
+    public static boolean canFociAppsConstructAccountsFromPrtIdTokens(@Nullable String negotiatedBrokerProtocol) {
```

Comment:
This method seems to be doing almost same thing as `canCompressBrokerPayloads`...should some of this code move to a helper method?

Reply:
done cb84a73eee5e8907dc89588ee566623e99383865